Couldn't build from sources on windows

Couldn't build from sources on windows:
## Steps to reproduce
Clone project into disk D:\ root and run build.cmd
## Expected  behaviour
Project built

## Actual behaviour
D:\core-setup>build.cmd
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.4.4
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************
Installing dotnet cli...
Restoring BuildTools version 2.0.0-prerelease-02215-01...
Initializing BuildTools...
Done initializing tools.
Running: D:\core-setup\Tools\msbuild.cmd /nologo /verbosity:minimal /clp:Summary /maxcpucount /nodeReuse:false /p:OSGroup=Windows_NT /p:TargetArchitecture=x64  /flp:v=normal  build.proj
MSBUILD : error MSB1009: Project file does not exist.
Switch: build.proj
Command execution failed with exit code 1.

## Environment data
Visual Studio 15.4.4
Windows 10

@danmosemsft 
```
D:\Work\core-setup>set _echo=1

D:\Work\core-setup>build.cmd
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.4.4
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************

C:\Users\mikha\source>setlocal

C:\Users\mikha\source>set INIT_TOOLS_LOG=D:\Work\core-setup\init-tools.log

C:\Users\mikha\source>if [] == [] set PACKAGES_DIR=D:\Work\core-setup\packages\

C:\Users\mikha\source>if [] == [] set TOOLRUNTIME_DIR=D:\Work\core-setup\Tools

C:\Users\mikha\source>set DOTNET_PATH=D:\Work\core-setup\Tools\dotnetcli\

C:\Users\mikha\source>if [] == [] set DOTNET_CMD=D:\Work\core-setup\Tools\dotnetcli\dotnet.exe

C:\Users\mikha\source>if [] == [] set BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json

C:\Users\mikha\source>set /P BUILDTOOLS_VERSION= 0<"D:\Work\core-setup\BuildToolsVersion.txt"

C:\Users\mikha\source>set BUILD_TOOLS_PATH=D:\Work\core-setup\packages\Microsoft.DotNet.BuildTools\2.1.0-prerelease-02220-02\lib\

C:\Users\mikha\source>set INIT_TOOLS_RESTORE_PROJECT=D:\Work\core-setup\init-tools.msbuild

C:\Users\mikha\source>set BUILD_TOOLS_SEMAPHORE=D:\Work\core-setup\Tools\2.1.0-prerelease-02220-02\init-tools.completed

C:\Users\mikha\source>if [] == [force] (
if exist "D:\Work\core-setup\Tools" rmdir /S /Q "D:\Work\core-setup\Tools"
 if exist "D:\Work\core-setup\packages\Microsoft.DotNet.BuildTools" rmdir /S /Q "D:\Work\core-setup\packages\Microsoft.DotNet.BuildTools"
)

C:\Users\mikha\source>if exist "D:\Work\core-setup\Tools\2.1.0-prerelease-02220-02\init-tools.completed" (
echo Tools are already initialized.
 goto :EOF
)
Tools are already initialized.
Running: D:\Work\core-setup\Tools\msbuild.cmd /nologo /verbosity:minimal /clp:Summary /maxcpucount /nodeReuse:false /p:OSGroup=Windows_NT /p:TargetArchitecture=x64  /flp:v=normal  build.proj
MSBUILD : error MSB1009: Project file does not exist.
Switch: build.proj
Command execution failed with exit code 1.
```

@mfilippov 
I managed to repro your problem while building from windows command line. Temporary workaround until the problem is fixed is to use Visual Studio 2017 developer command prompt to build repo.
```
e:\src\ms\dotnet\core-setup>build
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.4.4
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************
Installing dotnet cli...
Restoring BuildTools version 2.1.0-prerelease-02220-02...
Initializing BuildTools...
Done initializing tools.
Running: E:\src\ms\dotnet\core-setup\Tools\msbuild.cmd /nologo /verbosity:minimal /clp:Summary /maxcpucount /nodeReuse:false /p:OSGroup=Windows_NT /p:TargetArchitecture=x64  /flp:v=normal  build.proj
MSBUILD : error MSB1009: Project file does not exist.
Switch: build.proj
Command execution failed with exit code 1.
```

--- a/clean.cmd
+++ b/clean.cmd
@@ -1,8 +1,20 @@
 @if not defined _echo @echo off
 setlocal EnableDelayedExpansion
 
-echo Stop VBCSCompiler.exe execution.
-for /f "tokens=2 delims=," %%F in ('tasklist /nh /fi "imagename eq VBCSCompiler.exe" /fo csv') do taskkill /f /PID %%~F
+set NO_DASHES_ARG=%1
+if not defined NO_DASHES_ARG goto no_help
+if /I [%NO_DASHES_ARG:-=%] == [?] goto Usage
+if /I [%NO_DASHES_ARG:-=%] == [h] goto Usage
+
+:no_help
+
+:: Check if VBCSCompiler.exe is running
+tasklist /fi "imagename eq VBCSCompiler.exe" |find ":" > nul
+:: Compiler is running if errorlevel == 1
+if errorlevel 1 (
+	echo Stop VBCSCompiler.exe execution.
+	for /f "tokens=2 delims=," %%F in ('tasklist /nh /fi "imagename eq VBCSCompiler.exe" /fo csv') do taskkill /f /PID %%~F
+)
 
 :: Strip all dashes off the argument and use invariant
 :: compare to match as many versions of "all" that we can

--- a/run.cmd
+++ b/run.cmd
@@ -9,11 +9,12 @@ if exist %_VSWHERE% (
 )
 if not exist "%_VSCOMNTOOLS%" set _VSCOMNTOOLS=%VS140COMNTOOLS%
 if not exist "%_VSCOMNTOOLS%" (
-  echo Error: Visual Studio 2015 or 2017 required.
-  echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
-  exit /b 1
+    echo Error: Visual Studio 2015 or 2017 required.
+    echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
+    exit /b 1
 )
 
+set VSCMD_START_DIR="%~dp0"
 call "%_VSCOMNTOOLS%\VsDevCmd.bat"
 
 :Run

--- a/src/corehost/build.cmd
+++ b/src/corehost/build.cmd
@@ -51,6 +51,7 @@ if exist %_VSWHERE% (
 if not exist "%_VSCOMNTOOLS%" set _VSCOMNTOOLS=%VS140COMNTOOLS%
 if not exist "%_VSCOMNTOOLS%" goto :MissingVersion
 
+set VSCMD_START_DIR="%~dp0"
 call "%_VSCOMNTOOLS%\VsDevCmd.bat"
 
 :RunVCVars

--- a/src/uwp/build.cmd
+++ b/src/uwp/build.cmd
@@ -42,6 +42,7 @@ if exist %_VSWHERE% (
 if not exist "%_VSCOMNTOOLS%" set _VSCOMNTOOLS=%VS140COMNTOOLS%
 if not exist "%_VSCOMNTOOLS%" goto :MissingVersion
 
+set VSCMD_START_DIR="%~dp0"
 call "%_VSCOMNTOOLS%\VsDevCmd.bat"
 
 :RunVCVars

